rust/noosphere-ucan/README.md

```diff
@@ -0,0 +1,13 @@
+![API Stability: Alpha](https://img.shields.io/badge/API%20Stability-Alpha-red)
+
+# Noosphere Ucan
```

Comment:
Can you leave a comment here regarding the `v0.10.0 -> v1.0.0` major breaking changes as a rationale for the fork? Something like: "This project was forked from the official `rs-ucan` implementation, pending the finalization and broad adoption of the v1.0.0 spec (which represents a major breaking change from v0.10.0)."